### PR TITLE
Remove axios.all() and axios.spread() from Readme.md

### PR DESCRIPTION
**Pull request by ohtaeg**
- remove axios.all(), axios.spread()
- this is related issue #2260

thanks :)

### CODE REVIEW

**Comment by yasuf** [comment]
can you maybe do what the comment here suggested? https://github.com/axios/axios/issues/1042#issuecomment-331470391

**Comment by chinesedfan** [comment]
Instead of removing them directly, I recommend to mark them as deprecated first and remove in the next version.

**Comment by ohtaeg** [comment]
@yasuf  @chinesedfan 
Thank you for your kind explanation.
I changed it as you requested.
Please let me know if you need any modifications.
Thanks :)

File: README.md [hunk 1]
```diff
@@ -141,10 +141,10 @@ function getUserPermissions() {
   return axios.get('/user/12345/permissions');
 }
 
-axios.all([getUserAccount(), getUserPermissions()])
-  .then(axios.spread(function (acct, perms) {
+Promise.all([getUserAccount(), getUserPermissions()])
+  .then(function (acct, perms) {
```
Comment (chinesedfan) [inline README.md]: It should be,

```
function (results) {
  const acct = results[0];
  const perm = results[1];
}
```

File: README.md [hunk 2]
```diff
@@ -200,12 +200,12 @@ For convenience aliases have been provided for all supported request methods.
 ###### NOTE
 When using the alias methods `url`, `method`, and `data` properties don't need to be specified in config.
 
-### Concurrency
+~~Concurrency~~
```
Comment (chinesedfan) [inline README.md]: Better to add a sentence here to explain why they are removed. Say,

```
Deprecated. Use `Promise.all` to replace them.
```

Comment (jasonsaayman) [reply README.md]: Mended

File: README.md (hunk 1 above)
Comment (jasonsaayman) [reply README.md]: Mended

**Pull request merged**